source/wifi/wifi_hal.c

```diff
@@ -9025,11 +9138,11 @@ INT wifi_getRadioVapInfoMap(wifi_radio_index_t index, wifi_vap_info_map_t *map)
     map->num_vaps = 3; // XXX: this is a hack. For both radio let's support 3 vaps for now
                        //      home, backhaul and onboard.
     WIFI_ENTRY_EXIT_DEBUG("Inside %s:%d\n",__func__, __LINE__);
-    printf("Entering %s index = %d", __func__, (int)index);
+    printf("Entering %s radioIndex = %u", __func__, radioIndex);
```

Comment:
I'm aware that this is just log content change but can we also change printf to wifi_dbg_printf?